securitas/templates/group.html

```diff
@@ -1,125 +1,101 @@
-            </div>
-          </div>
-        </form>
-      </div>
-    </div>
-  {% endif %}
-
   <div class="container section">
-
     <div class="row" data-section="identity">
-      <div class="col s12">
+      <div class="col">
         <h3 class="header light-blue-text text-darken-4 fw300">{{ group.name }}</h3>
         {% if group.description %}<p>{{ group.description }}</p>{% endif %}
       </div>
     </div>
-
     <div class="row">
-      <div class="col m6 s12" data-section="sponsors">
-        <h4 class="header fw300">Sponsors</h4>
+      <div class="col-12" data-section="sponsors">
+        <h4>Sponsors</h4>
         {% if sponsors|length == 0 %}
         <p>No sponsors yet.</p>
         {% else %}
-        <ul class="collection">
+        <div class="row">
           {% for sponsor in sponsors|sort(attribute='username') %}
-            <li class="collection-item avatar">
-              <!-- TODO: .red if sponsor/admin -->
-              {% if sponsor.mail %}
-                <img src="{{ gravatar(sponsor.mail, 200) }}" class="circle">
-              {% else %}
-                <i class="material-icons circle">person</i>
-              {% endif %}
-              <a href="{{ url_for('user', username=sponsor.username) }}">
-                <span class="title">{{ sponsor.username }}</span>
-              </a>
-              <p>{{ sponsor.name }}</p>
-              <p>sponsor</p>
-            </li>
+            <div class="col-4 mb-4">
+              <div class="card">
+              <div class="media align-items-center card-body">
+                {% if sponsor.mail %}
+                  <img src="{{ gravatar(sponsor.mail, 80) }}">
+                {% else %}
+                  <i class="fa fa-user fa-2x text-muted mr-3"></i>
+                {% endif %}
+                <div class="media-body">
+                <a href="{{ url_for('user', username=sponsor.username) }}">
+                  {{ sponsor.username }}
+                </a>
+                <div>{{ sponsor.name }}</div>
+                </div>
+              </div>
+              </div>
+            </div>
           {% endfor %}
-        </ul>
+        </div>
         {% endif %}
       </div>
-
-      <div class="col m6 s12" data-section="members">
-        <h4 class="header fw300">
-          Users
-          {% if current_user_is_sponsor %}
-            <button class="btn btn-floating btn-small right waves-effect waves-light red modal-trigger" data-target="sponsor-modal">
-              <i class="material-icons">add</i>
-            </button>
+      <div class="col-12" data-section="members">
+        <div class="d-flex justify-content-between">
+          <div class="h4">Members</div>
+          {% if current_user_is_sponsor and sponsor_form %}
+          {# TODO: not require a double enter here maybe #}
+             <form action="{{ url_for('group_add_member', groupname=group.name) }}" method="post">
```

Comment:
Maybe this form could use some bottom margin? It's touching the border of the member list

Reply:
make the text box the small size ('form-control-sm') to provide a little more space at the bottom.
change made in:
https://github.com/fedora-infra/securitas/pull/92/commits/63a03bf4e8a20b90fb08165c89114903ab14b8a1